it/v2/connectors/ValidationConnectorSpec.scala

```diff
@@ -322,6 +326,92 @@ class ValidationConnectorSpec
 
     }
 
+    "When validating large messages" - {
+
+      "then on successful validation of schema valid XML, must return NO_CONTENT" in {
+        server.stubFor(
+          post(
+            urlEqualTo("/transit-movements-validator/messages/IE015/validation")
+          )
+            .withHeader(XObjectStoreUriHeader, equalTo("resource/location"))
+            .willReturn(
+              aResponse().withStatus(NO_CONTENT)
+            )
+        )
+        implicit val hc: HeaderCarrier = HeaderCarrier(extraHeaders = Seq(HeaderNames.ACCEPT -> ContentTypes.JSON))
```

Comment:
I think Sandeep meant this line -- the connector should handle ensuring the headers are correct so putting this here effectively bypasses that, and the test wouldn't catch it if this was missed.

Reply:
Yes, get it now!